frontend/components/FrequentQuestions.vue

```diff
@@ -0,0 +1,57 @@
+<script lang="ts" setup>
+import { mockData } from "~/shared/mockData";
+
+interface Props {
+  questionsGroup: string;
+  title: string;
+}
+const props = defineProps<Props>();
+
+const questionAnswers: Map<string, string> | undefined =
+  mockData.getQuestionAnswers(props.questionsGroup);
+
+if (questionAnswers === undefined) {
+  console.log(
+    "undefined questionGroup found for the frequentQuestions component",
+  );
+}
+</script>
+
+<template>
+  <div class="mw-100">
+    <h2 class="uu-sidebar-header-linked">
+      {{ $t(title) }}
+    </h2>
+    <div class="accordion mw-100" :id="'accordion' + questionsGroup">
+      <div
+        class="mw-100"
+        v-for="(questionAnswer, index) in questionAnswers"
+        :key="questionsGroup + index"
+      >
+        <div class="accordion-item mw-100">
+          <h2 class="accordion-header">
+            <button
+              class="accordion-button"
+              type="button"
+              data-bs-toggle="collapse"
+              v-bind:data-bs-target="'#collapse' + questionsGroup + index"
```

Comment:
Please also remove `v-bind` elsewhere in this component.


```suggestion
              :data-bs-target="'#collapse' + questionsGroup + index"
```